Server/routes/Mobile/rent.js

```diff
@@ -1,10 +1,140 @@
 const express = require('express')
+const PDFDocument = require('pdfkit');
 
 const router = express.Router()
 const passport = require('passport')
 const rentCtrl = require("../../controllers/Mobile/rent")
 const userCtrl = require("../../controllers/Mobile/user")
 const itemCtrl = require("../../controllers/Common/items")
+const transporter = require('../../middleware/transporter')
+const containerCtrl = require('../../controllers/Common/container')
+
+function formatDate(date) {
```

Comment:
Can be used in other file, create a specific file for it that can be imported from anywhere